src/__test__/server.ts

```diff
@@ -0,0 +1,70 @@
+import * as http from "http";
+import express from 'express';
+import { newEnforcer, Enforcer } from 'casbin';
+
+class CasbinService {
+    private enforcer! : Enforcer;
+    
+    public async run() {
+        // RBAC API doesn't support RBAC w/ domain.
+        // this.enforcer = await newEnforcer('./src/__test__/example/rbac_with_domains_model.conf', './src/__test__/example/rbac_with_domains_policy.csv');
+        this.enforcer = await newEnforcer('./src/__test__/example/rbac_model.conf', './src/__test__/example/rbac_policy.csv');
+    }
+    
+    public async getProfiles(sub: string) : Promise<string> {
+        const policies = await this.enforcer.getImplicitPermissionsForUser(sub);
+        let profiles : { [key:string]: string[] } = {};
```

Comment:
I think it's better to use `Map`.

Reply:
Yep, I agree with that

Reply:
After some consideration, I believe that use the raw object type `{ [key:string]: string[] }` may be more easy to manipulate the change on the objects' value. If we use Map<string, string[]> here, I think the code could be quite lengthy:
```typescript
let profiles : Map<string, string[]> = new Map<string, string[]>();
// Firstly, check if the key(action) exists
if (!profiles.has(action)) {
    profiles.set(action, []);
}
// Then take out the string array from the map, push new object inside, and set it back to the map
let arr : string[] = profiles.get(action);
arr.push(another_object);
profiles.set(action, arr);
// We need to pass the data to the frontend, so the map needs to be convert to JSON string
// However, apply JSON.stringify on a Map<string, string> instance doesn't work properly
// We need to convert Map<string, string> to object before using JSON.stringify
let obj : {[key: string]: string[]} = {};
profiles.forEach((value, key) => (obj[key] = value));
return JSON.stringify(obj);
```

In comparison, using `{ [key:string]: string[] }` here can be cleaner and more readable.

Reply:
This is a simple implementation of generating user profiles at the backend. I wish to add this api to the Casbin/node-casbin library, which can ease the process of getting a profiles of a given user. The implementation details can be discussed later.

Reply:
okay, keep the `{ [key:string]: string[] }`.